toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/logic/test_xnvme_tests_cli.py

```diff
@@ -0,0 +1,21 @@
+from ..conftest import xnvme_parametrize
+
+
+def test_optional_none(cijoe):
+
+    err, _ = cijoe.run("xnvme_tests_cli optional")
+    assert not err
+
+
+@xnvme_parametrize(labels=[], opts=["be", "mem", "sync", "async", "admin"])
+def test_optional_all(cijoe, device, be_opts, cli_args):
+
+    err, _ = cijoe.run(
+        f"xnvme_tests_cli optional "
```

Comment:
It is not needed here.
The unused ``cli_args`` is not pretty, however, it is provided since ``xnvme_parametrize()`` produces the list of ``(device, be_opts, cli_args)``, if the argument is removed from the test-func, then it fails.